packages/autocomplete-core/src/types/AutocompleteOptions.ts

```diff
@@ -35,95 +35,134 @@ export type InternalGetSources<TItem extends BaseItem> = (
 
 interface OnStateChangeProps<TItem extends BaseItem>
   extends AutocompleteScopeApi<TItem> {
+  /**
+   * The current Autocomplete state.
+   */
   state: AutocompleteState<TItem>;
+  /**
+   * The previous Autocomplete state.
+   */
   prevState: AutocompleteState<TItem>;
 }
 
 export interface AutocompleteOptions<TItem extends BaseItem> {
   /**
-   * Whether to consider the experience in debug mode.
+   * A flag to activate the debug mode.
+   *
+   * This is useful while developing because it keeps the panel open even when the blur event occurs. **Make sure to disable it in production.**
    *
-   * The debug mode is useful when developing because it doesn't close
-   * the panel when the blur event occurs.
+   * See [**Debugging**](https://autocomplete.algolia.com/docs/debugging) for more information.
    *
    * @default false
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#debug
    */
   debug?: boolean;
   /**
-   * The unique Autocomplete ID to create accessible attributes.
+   * An ID for the autocomplete to create accessible attributes.
    *
    * It is incremented by default when creating a new Autocomplete instance.
    *
    * @default "autocomplete-0"
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#id
    */
   id?: string;
   /**
-   * Function called when the internal state changes.
+   * The function called when the internal state changes.
+   *
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#onstatechange
    */
   onStateChange?(props: OnStateChangeProps<TItem>): void;
   /**
-   * The text that appears in the search box input when there is no query.
+   * The placeholder text to show in the search input when there's no query.
+   *
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#placeholder
    */
   placeholder?: string;
   /**
-   * Whether to focus the search box when the page is loaded.
+   * Whether to focus the search input or not when the page is loaded.
    *
    * @default false
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#autofocus
    */
   autoFocus?: boolean;
   /**
    * The default item index to pre-select.
    *
+   * We recommend using `0` when the query typed aims at opening item links, without triggering an actual search.
+   *
    * @default null
+   * @link https://autocomplete.algolia.com/docs/autocomplete-js#defaultactiveitemid
    */
   defaultActiveItemId?: number | null;
   /**
-   * Whether to open the panel on focus when there's no query.
+   * Whether to open the panel on focus or not when there's no query.
```

Comment:
```suggestion
   * Whether to open the panel on focus when there's no query.
```

previous description is less ambiguous imo

Reply:
I agree. I copied the official doc. I'll update both.